### ISSUE TITLE
Deep Learning for Computational Chemistry

### ISSUE THREAD

**Issue by swamidass** [greenelab/deep-review#251]
https://arxiv.org/abs/1701.04503

Another review paper that needs to be carefully reviewed and added.

**Comment by swamidass** [comment]
This review is fortunate for us @agitter. I think we can offload some of the heavy work by pointing to this as a more comprehensive review of chemistry related work. And say we are focusing on some of the key highlights. That will help us reduce scope to something a bit more reasonable for this very far ranging review.

What do you think?

**Comment by agitter** [comment]
Yes, I think that is a good idea.  We can focus on the message we want to get across in this paper and not be as concerned with covering all related work.
